ui/site/src/site.ts

```diff
@@ -104,6 +104,10 @@ lichess.load.then(() => {
     $('input.confirm, button.confirm').on('click', function (this: HTMLElement) {
       return confirm(this.title || 'Confirm this action?');
     });
+    $('input.confirm, button.alert-false').on('click', function (this: HTMLElement) {
```

Comment:
this adds the behaviour to all `input.confirm` on every page.

It also looks for `button.alert-false` on every page.\

Reply:
```
      (res.n == 1) ?? {
         teamRepo.incMembers(team.id, -1) >>
           (team.leaders contains userId) ?? teamRepo.setLeaders(team.id, team.leaders - userId)
       } >>-
```
Glad you spotted that.  Totally overlooked that incMembers was a future